context-monitoring/src/test/java/de/atb/context/services/TestMonitoringDataRepositoryService.java

```diff
@@ -76,8 +76,8 @@ public static void beforeClass() {
         props.setProperty("org.apache.cxf.stax.allowInsecureParser", "true");
 
 		String absolutefilePath = new File("").getAbsolutePath();
-		configFile = new File(
-				absolutefilePath.concat(File.separator + "resources"+ File.separator + "services-config.xml"));
+        configFile = new File(
+            absolutefilePath.concat(File.separator + "src" + File.separator + "test" + File.separator + "resources" + File.separator + "services-config.xml"));
```

Comment:
`String filePath = Path.of("src", "test", "resources", "services-config.xml").toAbsolutePath().toString()`

Reply:
done